src/main.ts

```diff
@@ -308,6 +331,62 @@ export default class MediaDbPlugin extends Plugin {
 		return fileContent;
```

Comment:
I would add the templater media object here as well if the templater setting is enabled.